### PR TITLE
Add Flathub manifest for kontrast

### PR DESCRIPTION
**Pull request by ognarb**
Kontrast is a new KDE application developed by myself.

https://invent.kde.org/accessibility/kontrast

### CODE REVIEW

**Comment by tsdgeos** [comment]
bot, build org.kde.kontrast

File: org.kde.kontrast.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+id: org.kde.kontrast
+runtime: org.kde.Platform
+runtime-version: "5.15"
+sdk: org.kde.Sdk
+command: kontrast
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11"
```
Comment (barthalion) [inline org.kde.kontrast.yaml:8]: It should be `fallback-x11` if it works natively on Wayland.

**Comment by barthalion** [comment]
bot, build org.kde.kontrast

**Comment by ognarb** [comment]
bot, build org.kde.kontrast

File: org.kde.kontrast.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+id: org.kde.kontrast
+runtime: org.kde.Platform
+runtime-version: "5.15"
+sdk: org.kde.Sdk
+command: kontrast
+finish-args:
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
```
Comment (tsdgeos) [inline org.kde.kontrast.yaml:9]: I think we need "--device=dri",
to get away from that ugly warning when starting

libGL error: MESA-LOADER: failed to retrieve device information
libGL error: Version 4 or later of flush extension not found
libGL error: failed to load driver: i915
libGL error: failed to open /dev/dri/card0: El fitxer o directori no existeix
libGL error: failed to load driver: i965

**Comment by tsdgeos** [comment]
Can you please switch this to json? All our other KDE manifests are in json so we want to maintain consistency so that we can eventually automate stuff (i have some awful scripts that do this)

**Comment by ognarb** [comment]
> Can you please switch this to json? All our other KDE manifests are in json so we want to maintain consistency so that we can eventually automate stuff (i have some awful scripts that do this)

Ok, changed

**Comment by tsdgeos** [comment]
bot, build org.kde.kontrast

**Comment by tsdgeos** [comment]
Let's rebuild again just in case :)

**Comment by tsdgeos** [comment]
Seems reasonable to me

**Pull request closed without merging**